plugins/module_utils/network/sonic/config/mclag/mclag.py

```diff
@@ -193,6 +200,51 @@ def _state_deleted(self, want, have):
                     commands = update_states(diff_want, "deleted")
         return commands, requests
 
+    def _state_replaced(self, want, have, diff):
+        """ The command generator when state is merged
+
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        requests = []
+        commands = []
+        if diff:
+            requests = self.get_create_mclag_request(want, diff)
+            if len(requests) > 0:
+                commands = update_states(diff, "replaced")
+        return commands, requests
+
+    def _state_overridden(self, want, have, diff):
+        """ The command generator when state is overridden
+        :param want: the desired configuration as a dictionary
+        :param obj_in_have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        requests = []
+
+        if not diff:
+            return commands, requests
+
```

Comment:
This may not be correct. When want=c, and before have=ABC (c is part of C), after have should be have=c. But this code makes no change. Please verify.